extensions/timeular/src/useTodayReport.ts

```diff
@@ -23,18 +23,37 @@ export const useTodayReport = (tracking: Tracking | null, activity?: Activity) =
     if (reportIsLoading) {
       return;
     }
+    const ms = Date.now() - new Date(tracking?.startedAt + "Z").getTime();
 
     const markStart = Date.now();
 
+    const totalBookedTime = entries.reduce((acc, val) => {
+      if (!val.duration.stoppedAt || !val.duration.startedAt) {
+        // Skip entries that are still running (no stoppedAt)
+        return acc;
+      }
+
+      const [startedAt, stoppedAt] = [
+        new Date(val.duration.startedAt + "Z").getTime(),
+        new Date(val.duration.stoppedAt + "Z").getTime(),
+      ];
+
+      return (acc += stoppedAt - startedAt);
+    }, 0);
+
     Promise.resolve(tracking && activity ? entries.concat([mockEntry(tracking, activity)]) : entries)
       .then(entries => entries.reduce(groupByActivity, [] as TimeEntry[][]))
-      .then(groupped => groupped.map(arr => arr.reduce(groupIntoReport, blankReport(arr[0].activity))))
+      .then(grouped => grouped.map(arr => arr.reduce(groupIntoReport, blankReport(arr[0].activity))))
       .then(entries => entries.sort((l, r) => (l.duration > r.duration ? -1 : 1)))
       .then(entries => entries.map(entry => `* ${entry.activity.name} for ${humanizeDuration(entry.duration)}`))
       .then(list => list.join("\n\n"))
       .then(markdown =>
         markdown
-          ? setReportMarkdown("Time entries you have tracked today:\n" + markdown)
+          ? setReportMarkdown(
+              `Time entries you have tracked today (including current)[**${humanizeDuration(
+                totalBookedTime + (ms || 0)
+              )}**]:\n` + markdown
+            )
```

Comment:
I would suggest converting this syntax into a basic anonymous function (`() => {}`), extracting things to consts, etc., to make this thing easier to read&comprehend.

Also, what do you think about this word change: switch from `[HH:mm]` to `... is HH:mm`, so it reads as:
```
Time entries you have tracked today (including current) is 3h:14m: # time can be bold if
- ...                                                              # that works better
                                                                   # for you
```

Reply:
I like that suggestion. I have extracted it to `buildHeadlineWithTotal`. I utilized a useCallback. 

I like the content suggestion as well.